resources/aws/eks/nodeGroups.go

```diff
@@ -27,15 +27,25 @@ const (
 )
 
 func NewLinuxNodeGroup(e aws.Environment, cluster *eks.Cluster, nodeRole *awsIam.Role) (*eks.ManagedNodeGroup, error) {
-	return newManagedNodeGroup(e, "linux-ng", cluster, nodeRole, amazonLinux2AMD64AmiType, e.DefaultInstanceType())
+	return newManagedNodeGroup(e, "linux", cluster, nodeRole, amazonLinux2AMD64AmiType, e.DefaultInstanceType())
 }
 
 func NewLinuxARMNodeGroup(e aws.Environment, cluster *eks.Cluster, nodeRole *awsIam.Role) (*eks.ManagedNodeGroup, error) {
-	return newManagedNodeGroup(e, "linux-arm-ng", cluster, nodeRole, amazonLinux2ARM64AmiType, e.DefaultARMInstanceType())
+	return newManagedNodeGroup(e, "linux-arm", cluster, nodeRole, amazonLinux2ARM64AmiType, e.DefaultARMInstanceType())
 }
 
 func NewBottlerocketNodeGroup(e aws.Environment, cluster *eks.Cluster, nodeRole *awsIam.Role) (*eks.ManagedNodeGroup, error) {
-	return newManagedNodeGroup(e, "bottlerocket-ng", cluster, nodeRole, bottlerocketAmiType, e.DefaultInstanceType())
+	return newManagedNodeGroup(e, "bottlerocket", cluster, nodeRole, bottlerocketAmiType, e.DefaultInstanceType())
+}
+
+func nodeGroupNamePrefix(stack, name string) string {
+	prefix := stack + "-" + name + "-ng"
+	nbExceeding := len(prefix) - 37
+	if nbExceeding <= 0 {
+		return prefix
+	}
+
+	return stack[:len(stack)-nbExceeding/2] + "-" + name[:len(name)-nbExceeding/2-nbExceeding%2] + "-ng"
```

Comment:
I don't think there is a guaranty that `nbExceeding/2` < `len(stack)` or `nbExceeding/2+nbExceeding%2 <  len(name)`

Reply:
Good catch.
I think that, if I want to truncate both the `stack` and `name` strings, it needs to be done proportionally to the original size of the strings: https://github.com/DataDog/test-infra-definitions/pull/216/commits/09ce545b5d15e266062341fed36031afb7ff3927